web3.js/src/address-lookup-table-program.ts

```diff
@@ -169,17 +171,16 @@ export class AddressLookupTableInstruction {
 
   static decodeExtendLookupTable(
     instruction: TransactionInstruction,
-    numberOfAddresses: number,
   ): ExtendLookupTableParams {
     this.checkProgramId(instruction.programId);
-    if (instruction.keys.length < 2 || instruction.keys.length > 4) {
```

Comment:
Note that I removed this constraint because it's fine if extra keys are appended to the instruction, they will just be ignored.